opm-test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
```

Comment:
I think we should `set -euxo pipefail` here
https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/